test/sqllogictest/interval.slt

```diff
@@ -806,8 +806,66 @@ SELECT INTERVAL '1.0 second 42 milliseconds'
 ----
 00:00:01.042
 
-statement error Cannot set MILLISECONDS field if SECOND field has a fraction component
+query T
+SELECT INTERVAL '5 microseconds'
+----
+00:00:00.000005
+
+query T
+SELECT INTERVAL '5000006 microseconds'
```

Comment:
Looks like this is valid, as well:

```
SELECT INTERVAL '5000006 microseconds 1 second';
    interval
-----------------
 00:00:06.000006
```

would like to see this added just to ensure the blockage of setting values only goes from fractional seconds to microseconds and not the other way around.